src/modules/assets/assets.service.ts

```diff
@@ -351,7 +351,9 @@ export class AssetsService {
       return {
         identity: parsedLog.args.identity,
         owner: parsedLog.args.owner,
-        at: parsedLog.args.at,
+        at:
```

Comment:
This should work too. Another option is https://github.com/energywebfoundation/ew-credentials/blob/0f3796bce1a7c00ddc27a979cb94581aa5984eea/packages/credential-governance/src/domain-hierarchy.ts#L241

Reply:
I tried your solution, but the same issue.